keywords.txt

```diff
@@ -39,6 +39,7 @@ STANDARD	LITERAL1
 INVERTED	LITERAL1
 almostEqual	KEYWORD2
 cruiseControlEnabled	KEYWORD2
+disableControlEnabled	KEYWORD2
 LEFT	LITERAL1
```

Comment:
The name of the function is `disableCruiseControl` so please modify accordingly. :)